GitUI/CommandsDialogs/RefLogDialog/BranchNameForm.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Windows.Forms;
+
+namespace GitUI.CommandsDialogs.RefLogDialog
+{
+    public partial class BranchNameForm : Form
```

Comment:
Thanks! I tested it and I find it a better solution. The existing Create Branch dialog is well known to users so they should have no problem with the additional complexity. I would only uncheck the Checkout After Create checkbox when using from the RefLogDialog.